packages/core/breadcrumb.js

```diff
@@ -1,21 +1,21 @@
 const { isoDate } = require('./lib/es-utils')
 
-class BugsnagBreadcrumb {
-  constructor (name = '[anonymous]', metaData = {}, type = 'manual', timestamp = isoDate()) {
+class Breadcrumb {
+  constructor (message = '[empty]', metadata = {}, type = 'manual', timestamp = isoDate()) {
```

Comment:
No, it's not validated. The types prevent TypeScript users from supplying an invalid type, but  for regular JS users, the `client.leaveBreadcrumb()` method just ensures that it is a string.

I think the result of providing an invalid value would be that the pipeline drops it (rather than coercing the type to `"manual"`), so I will confirm that first, then see if we can provide better validation without increasing the bundle size (too much).